prow/scripts/lib/cli-alpha.sh

```diff
@@ -3,11 +3,13 @@
 # cli-alpha::deploy starts Kyma installation using the alpha deploy command
 #
 # Arguments:
-#	$1 - Path to local resource directory
-#	$2 - Path to local components.yaml file
+#	$1 - OPtional path to an overrides file
 function cli-alpha::deploy {
+	# shellcheck disable=SC2119
+	local overrides=$1
```

Comment:
This should pass shellcheck
```suggestion
#	$1 - Optional path to an overrides file
# shellcheck disable=SC2119
function cli-alpha::deploy {
	local overrides=$1
```

Reply:
Thanks a lot @Halamix2 for the hint with the shellcheck! But I finally removed the "smelly" line complete, as we have currently no real needs in the tests for it :)